### ISSUE TITLE
npm install fail (windows)

### ISSUE THREAD

**Issue by ami44** [FormidableLabs/spectacle-boilerplate#3]
(windows) npm instal return error 
```
npm ERR! peerinvalid The package history@1.13.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-router@1.0.3 wants history@^1.17.0
```

full log
```
λ npm install
npm WARN package.json spectacle-boilerplate@1.0.0 No repository field.
npm WARN optional dep failed, continuing fsevents@1.0.6
npm WARN optional dep failed, continuing fsevents@1.0.6
npm WARN peerDependencies The peer dependency history@^1.17.0 included from react-router will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN optional dep failed, continuing fsevents@1.0.6

> react-router@1.0.3 postinstall C:\ami44\spectacle-boilerplate\node_modules\spectacle\node_modules\react-router
> node ./npm-scripts/postinstall.js

/
> caniuse-api@1.4.1 postinstall C:\ami44\spectacle-boilerplate\node_modules\spectacle\node_modules\radium\node_modules\inline-style-prefixer\node_modules\caniuse-api
> node generator.js

caniuse-api: Generation ok

> radium@0.15.0 postinstall C:\ami44\spectacle-boilerplate\node_modules\spectacle\node_modules\radium
> node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || npm run lib

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "D:\\Nodist\\bin\\v-x64\\nodev4.2.2\\node.exe" "D:\\Nodist\\bin\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.10
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package history@1.13.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-router@1.0.3 wants history@^1.17.0

npm ERR! Please include the following file with any support request:
npm ERR!     C:\ami44\spectacle-boilerplate\npm-debug.log

```

**Comment by knowbody** [comment]
@ami44 I've just tried installing `spectacle-boilerplate` and works just fine. Make sure that you:

```
rm -rf node_modules && npm i
```

I've checked both on npm v2 and v3

**Comment by kenwheeler** [comment]
@knowbody I just fixed that with spectacle 1.0.3., this can get closed

**Comment by knowbody** [comment]
:+1: thanks!
